.gitignore

```diff
@@ -5,8 +5,8 @@
 # Log file
 *.log
 
-#conf file
-ballerina.conf
+#conf file (Commented to add ballerina.conf)
```

Comment:
Do we need to commit the conf file? Within the connector implementation, we are not using the conf file right? It's the user of the connector who needs to use the conf. Therefore we can put the conf details in guides and in the examples in module md if needed.

Reply:
Fixed.